streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -353,26 +353,6 @@ public static WindowBytesStoreSupplier inMemoryWindowStore(final String name,
         return new InMemoryWindowBytesStoreSupplier(name, retentionMs, windowSizeMs, retainDuplicates);
     }
 
-    /**
-     * Create an in-memory {@link SessionBytesStoreSupplier}.
-     * @param name              name of the store (cannot be {@code null})
-     * @param retentionPeriod   length ot time to retain data in the store (cannot be negative)
-     *                          Note that the retention period must be at least long enough to contain the
-     *                          windowed data's entire life cycle, from window-start through window-end,
-     *                          and for the entire grace period.
-     * @return an instance of a {@link  SessionBytesStoreSupplier}
-     */
-    public static SessionBytesStoreSupplier inMemorySessionStore(final String name, final Duration retentionPeriod) {
```

Comment:
Just reordering to "correct" position in alignment to the order of the other method.